src/ICS20Transfer.sol

```diff
@@ -23,7 +24,10 @@ using SafeERC20 for IERC20;
  */
 contract ICS20Transfer is IIBCApp, IICS20Transfer, IICS20Errors, Ownable, ReentrancyGuard {
     /// @notice Mapping of non-native denoms to their respective IBCERC20 contracts created here
-    mapping(string denom => IBCERC20 ibcERC20Contract) private _foreignDenomContracts;
+    mapping(string denom => IBCERC20 ibcERC20Contract) private _ibcDenomContracts;
+
+    /// @notice Mapping of channel ID to escrow contract address
+    mapping(string channelId => ICS20Escrow ics20EscrowContract) private _channelEscrowContracts;
```

Comment:
I was thinking again about this and I think we only need the client Id on the ethereum side, since [when a client is created we set the counterparty info](https://github.com/cosmos/solidity-ibc-eureka/blob/ee85ba67606650fa9d38fc71dcaef6692c89defb/src/ICS02Client.sol#L66) (and there is no way to change it afterwards). So the counterparty client ID, client ID pair should remain fixed and immutable. Does this make sense?